README.md

```diff
@@ -1,26 +1,172 @@
+
+### Prerequisites
+
+You will need to obtain an API key to use Sindri.
+If you have not already created an account, follow the sign up link posted at the top left of this README.
+After logging into the [Sindri front-end](https://sindri.app/login), you can create and manage your API Keys within the [API Keys Settings page](https://sindri.app/z/me/page/settings/api-keys).
+
+You should also install `docker`, `kubectl`, `minikube`, `helm`, `node`, and `scrollsdk` as instructed by the official [Scroll SDK documentation](https://scroll-sdk-init.docs.scroll.xyz/en/sdk/guides/devnet-deployment/#prerequisites).
+
+At this point, you should start `minikube` via
+```
+minikube start --driver=docker
+```
+
+### Obtaining and Configuring Charts
+
+There is no need to work in this repository, as the guide will be utilizing publicly hosted images and Helm charts.
+However, you will need to clone the Scroll SDK repo and navigate to the `devnet/` directory to access some helper scripts.
+```bash
+git clone git@github.com:scroll-tech/scroll-sdk.git
+cd scroll-sdk/devnet
+```
+
+Next we will manually pull and extract the chart for the latest version of Scroll SDK.
+```bash
+make bootstrap
+```
+> ⚠️ **Encountering Permission Issues?**<br>
+> If you are given any prompts during the execution of the bootstrap command, then there is likely a root vs. user mismatch between configuration shell scripts and the downloaded files.
+> You can solve this by exiting the bootstrap process with `ctrl+c` and running the `config` command separately with `sudo make config`.
+
+
+### Installing the Helm Chart
+
+In this step, we will launch the sequencing layer, the coordinator, and various visibility services.
+Because provers require manual configuration, we will start those separately in a later section.
+Note that the standard devnet settings do not include any proving layer services, so you will need to add the following two lines to the end of the `install` command in `scroll-sdk/devnet/Makefile`.
+```bash
+    --set coordinator-api.enabled=true \
+    --set coordinator-cron.enabled=true
+```
+After that adjustment, enter `make install` in your terminal to start the chain.
+You can monitor the progress and health of all the services via the command `kubectl get pods`.
+
+> 🕒 **Waiting on the coordinator?**<br>
+> An essential functionality of the coordinator is the validation of proofs that are returned from a prover.
+> For this reason, there are two initial pods (`assets-download` and `parameter-download`) which must pull zkEVM circuit artifact files before the final `coordinator-api` pod starts.
+> You can watch the progress of the first pod with `kubectl logs coordinator-api-0 -c assets-download`.
+> Similarly, you can watch the progress of the trusted setup download via `kubectl logs coordinator-api-0 -c parameter-download`.
+
+### Accessing Services Locally
```

Comment:
These instructions assume that you are running the stack on a remote server and that you want to access that remote server from your local machine.

Should we modify these instructions to reflect the steps needed to run this on your local machine and access from your local machine?

Reply:
Agree with this suggestion. In fact, I was just running through these and they did not seem familiar.  Is it fair to just replace with the following?
```
minikube addons enable ingress
minikube addons enable ingress-dns
```
The `etc/hosts/` step was still required for me.  So that would be left alone, but the other two would get removed.

Do you have an easier route though?

Reply:
I think that is correct - replace with minikube.

The /etc/hosts file still needs to be updated. But the IP address would be 127.0.0.1

Reply:
Addressed in https://github.com/Sindri-Labs/sindri-scroll-sdk/pull/8/commits/343d2047ed4ceb33637ce54bce762dc35e9ab272